iocore/net/OCSPStapling.cc

```diff
@@ -72,14 +73,18 @@ certinfo_map_free(void * /*parent*/, void *ptr, CRYPTO_EX_DATA * /*ad*/, int /*i
   }
 
   for (certinfo_map::iterator iter = map->begin(); iter != map->end(); ++iter) {
-    if (iter->second->uri) {
-      OPENSSL_free(iter->second->uri);
+    certinfo *cinf = iter->second;
+    if (cinf->uri) {
+      OPENSSL_free(cinf->uri);
     }
-    if (iter->second->certname) {
-      ats_free(iter->second->certname);
+    if (cinf->certname) {
```

Comment:
ats_free() already checks for nullptr

Reply:
Good to know. I'll make a follow up PR to clean all off those up!